framework/sandstone.cpp

```diff
@@ -1115,13 +1116,16 @@ Common command-line options are:
  -s <STATE>, --rng-state=<STATE>
      Specify the random generator state to reload. The seed is in the form:
        Engine:engine-specific-data
+ --random-compatibility-mode
+     Set compatibility mode of Random Number Generators (with very intense usage
```

Comment:
Need a clearer phrasing here. In particular, is the default behaviour same as it has always been? Or does this option turn on a new random mode?

Given the name, I suppose that the new default is to use the new technique, so the description should be "Enables generation of random floating-point numbers with random mantissa bits" or something similar.